config/types.go

```diff
@@ -9,12 +9,15 @@ type VM int
 const (
 	UNKNOWN_VM VM = iota
 	EVM
+	EVM_BLOCKHASH
```

Comment:
I might be misunderstanding something, but this doesn't feel like it should be a distinct VM, I think it would make more sense as a flag? I also don't really understand how we're using it right now - Does an `EVM_BLOCKHASH` subscriber perform a superset of the actions performed byan `EVM` subscriber?